File: fix_tail.py
import sys

path = r'd:\repositorio_geral\app_gestao_contratos\src\app\dashboard\templates\novo\page.tsx'

with open(path, 'r', encoding='utf-8') as f:
    content = f.read()

target = """            </div>

          </div>
        )}
 </button>
              </div>
            </div>

          </div>
        )}

      </div>"""

replacement = """            </div>

          </div>
        )}

      </div>"""

if target in content:
    content = content.replace(target, replacement)
elif """            </div>
 
           </div>
         )}
  </button>
               </div>
             </div>
 
           </div>
         )}
 
       </div>""" in content:
    content = content.replace("""            </div>
 
           </div>
         )}
  </button>
               </div>
             </div>
 
           </div>
         )}
 
       </div>""", """            </div>

          </div>
        )}

      </div>""")
else:
    print("Could not find exact string")
    sys.exit(1)

with open(path, 'w', encoding='utf-8') as f:
    f.write(content)

print("Success")

File: fix_tail.js
const fs = require('fs');
const path = require('path');

const filePath = path.join(__dirname, 'src', 'app', 'dashboard', 'templates', 'novo', 'page.tsx');
let content = fs.readFileSync(filePath, 'utf-8');

const target1 = `            </div>

          </div>
        )}
 </button>
              </div>
            </div>

          </div>
        )}`;

const target2 = `            </div>
 
           </div>
         )}
  </button>
               </div>
             </div>
 
           </div>
         )}`;

const replacement = `            </div>

          </div>
        )}`;

if (content.includes(target1)) {
    content = content.replace(target1, replacement);
    console.log("Replaced target1");
} else if (content.includes(target2)) {
    content = content.replace(target2, replacement);
    console.log("Replaced target2");
} else {
    console.log("Target not found exactly. Trying smaller tail cut.");
    // Let's just crop to where it ends or fix it manually with node
}

fs.writeFileSync(filePath, content, 'utf-8');
console.log("Done");
